benchmarks/test_gmg_free_surface/S20RTS.prm

```diff
@@ -0,0 +1,32 @@
+include $ASPECT_SOURCE_DIR/cookbooks/S20RTS.prm
```

Comment:
you will need to remove this folder again, I think.

Reply:
remove this prm? How about S20RTS_gmg.prm?

Reply:
You added the folder ``benchmarks/test_gmg_free_surface``. I think it doesn't  make sense to keep it around. It is not really a "benchmark". We could add some files next to existing ones (bla_gmg.prm next to bla.prm).

Reply:
I see. Do you want to keep the matrix-based version to make it easier to compare? (bla_amp.prm and bla_gmg.prm)
We changed some parameters in the matrix-based prm to get a better comparison with the matrix-free one.